backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/database/repos/CardRepository.kt

```diff
@@ -65,4 +74,37 @@ object CardRepository {
             card.firstActivationDate = Instant.now()
         }
     }
+
+    fun getCardStatisticsByProjectAndRegion(project: String, dateStart: Timestamp, dateEnd: Timestamp, regionId: Int?): List<CardStatisticsResultModel> {
+        val numAlias = Coalesce(Cards.id.count(), intLiteral(0)).alias("numCards")
+        val allCards = (Regions leftJoin Cards leftJoin Projects)
+            .slice(Regions.id, numAlias)
+            .select { Cards.issueDate.greaterEq(dateStart) and Cards.issueDate.lessEq(dateEnd) }
+            .groupBy(Regions.id)
+            .alias("AllCards")
+        val activeCards = (Regions leftJoin Cards)
+            .slice(Regions.id, numAlias)
+            .select { Cards.firstActivationDate.isNotNull() and Cards.issueDate.greaterEq(dateStart) and Cards.issueDate.lessEq(dateEnd) }
```

Comment:
Currently selection for "today" doesn't work, because cards have issue date with time, like "2024-04-30 11:45:11.248946", so the query condition like "WHERE (cards."issueDate" >= '2023-04-30T00:00:00') AND (cards."issueDate" <= '2024-04-30T00:00:00')" will produce no result.

As an option, issueDate could be cast to from timestamp to date before comparing (not sure which solution would make more sense from an implementation perspective).